### PR TITLE
Removes redundant check

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/modules/mob/living/living_defense.dm [hunk 1]
```diff
@@ -67,17 +67,15 @@
 		var/zone = ran_zone("chest", 65)//Hits a random part of the body, geared towards the chest
 		var/dtype = BRUTE
 		var/volume = I.get_volume_by_throwforce_and_or_w_class()
-		if(istype(I, /obj/item)) //If the item is a weapon...
```
Comment (RemieRichards) [inline code/modules/mob/living/living_defense.dm]: before someone reads this and goes "hurr durr" this used to say ```/obj/item/weapon``` but @Cyberboss 's regex killed it.

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/_globalvars/lists/maintenance_loot.dm#L91

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/crafting/guncrafting.dm

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/crafting/recipes.dm#L298

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/game/objects/items/stacks/sheets/sheet_types.dm#L135

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/research/designs/autolathe_designs.dm#L632

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/game/objects/structures.dm#L52

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/game/objects/structures/tables_racks.dm#L423

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/admin/verbs/debug.dm#L355

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/human/interactive.dm#L285
bunch of these might need updating.

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/human/interactive.dm#L636

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/human/interactive.dm#L1478
woo boy

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/human/interactive.dm#L1501

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/monkey/combat.dm#L391

not sure if this will need updating or anything.

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/carbon/monkey/combat.dm#L80

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/mob/living/living_defense.dm#L70

**Comment by ShizCalev** [comment]
3mzxp58
these need to be checked (mentioned above for the most part)

**Comment by Cyberboss** [comment]
We decided to not care about monkeycode and interactives (which are never used and getting removed eventually anyway)

**Comment by Shadowlight213** [comment]
I'm waiting on goap to refactor interactives and make them less buggy

**Pull request merged**